assets/video-camera-withBar.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   height="24px"
+   viewBox="0 0 24 24"
+   width="24px"
+   fill="#000000"
+   version="1.1"
+   id="svg38"
+   sodipodi:docname="video-camera-7.svg"
+   xml:space="preserve"
+   inkscape:version="1.2.1 (0f2f062aeb, 2022-09-21, custom)"
```

Comment:
Please save this file as `Plain SVG (*.svg)` (a file format option in Inkscape) instead of `Inkscape SVG (*.svg)`. In this case we don't need the metadata.

Please use lowercase for the file name to match the others: `video-camera-with-bar.svg`

Reply:
Thanks. No my branch does not touch the video camera icons.